### PR TITLE
Add more detailed test and benchmark instructions

### PR DESCRIPTION
**Pull request by meooow25**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,15 +22,33 @@ the `containers` package.
...
-cabal bench containers-test
+```
+
+Run all tests or benchmarks:
+```
+cabal test containers-tests
+cabal bench containers-tests
+```
+
+To run a particular test or benchmark suite, name the target from `containers-tests.cabal`:
```
Comment (treeowl) [inline CONTRIBUTING.md:36]:
```suggestion
To run a particular test or benchmark suite, name the target from `containers-tests/containers-tests.cabal`:
```

**Review by treeowl — CHANGES_REQUESTED** [review]
Awesome! I've requested one little change.

File: CONTRIBUTING.md (hunk 1 above)
Comment (meooow25) [reply CONTRIBUTING.md]: Done.

**Review by treeowl — APPROVED** [review]

**Comment by treeowl** [comment]
Thanks!

**Pull request merged**